src/status_im/ui/screens/chat/input/input.cljs

```diff
@@ -204,7 +204,7 @@
         (when input-text-empty?
           [stickers/button show-stickers?])
         (if input-text-empty?
-          [commands-button]
+          [react/view];;TODO temporay hide for v1 code freeze [commands-button]
```

Comment:
could it use the config/extensions-enabled? flag as well here?

Reply:
no, it's not related to extensions